cpp/src/arrow/ipc/options.h

```diff
@@ -94,6 +94,11 @@ struct ARROW_EXPORT IpcReadOptions {
   /// like decompression
   bool use_threads = true;
 
+  /// \brief Convert endian of data element to platform-native endianness
+  /// if the endianness of the received schema is not equal to
+  /// platform-native endianness
+  bool use_native_endian = true;
```

Comment:
Sorry. I suggested the `use_native_endian` name but it's not a good name...
It looks like that "if use_native_endian is false, the read data uses non native endian".

How about renaming it to `ensure_native_endian` or something?